aten/src/ATen/native/native_functions.yaml

```diff
@@ -3426,8 +3426,13 @@
   variants: function, method
 
 - func: pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: pixel_shuffle
+    QuantizedCPU: pixel_shuffle_quantized_cpu
 
 - func: pixel_unshuffle(Tensor self, int downscale_factor) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: pixel_unshuffle
```

Comment:
do we need this?

Reply:
If you want to be able to manually define the backward for it, yes.

Reply:
Ohhh, i see...